applications/luci-app-throughput-measurements/root/usr/share/rpcd/acl.d/luci-app-throughput-measurements.json

```diff
@@ -0,0 +1,17 @@
+{
+	"luci-app-throughput-measurements": {
+		"description": "Grant access to tputmeas ubus object",
+		"read": {
+			"cgi-io": [ "exec" ],
+			"uci": [ "tputmeas" ],
+			"ubus": {
+				"luci.tputmeas": [ "iperf3Start", "iperf3Stop" ],
+				"luci.tputmeas.notify": [ "*" ]
+			}
+		},
+		"write": {
+			"uci": [ "tputmeas" ],
+			"file": {"/usr/libexec/luci.tputmeas": ["exec"]}
```

Comment:
I think this must be a write ubus? or not

Reply:
It is not needed, I think